runners/spark/src/main/java/org/apache/beam/runners/spark/ExistingSparkContextOptions.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.beam.runners.spark;
+
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.PipelineOptions;
+
+/**
+ * Existing Spark Context pipeline options.
+ */
+public interface ExistingSparkContextOptions extends PipelineOptions {
+
+  /**
+   * Set to true if the spark runner will be
+   * initialized with an existing Spark Context.
+   */
+    @Description("Set to true if the spark runner will be "
+     + "initialized with an existing Spark Context")
+    boolean isProvidedJavaSparkContext();
```

Comment:
I wouldn't create this as a boolean option. but infer it from the JavaSparkContext presence if the user passes it is because it wants to use it.